File: app/src/main/java/moe/tlaster/openween/core/model/favor/FavorTagModel.java
package moe.tlaster.openween.core.model.favor;

import com.google.gson.annotations.SerializedName;

/**
 * Created by Tlaster on 2016/9/2.
 */
public class FavorTagModel {
    @SerializedName("id")
    private long mID;
    @SerializedName("tag")
    private String mTag;

    public long getID() {
        return mID;
    }

    public void setID(long ID) {
        mID = ID;
    }

    public String getTag() {
        return mTag;
    }

    public void setTag(String tag) {
        mTag = tag;
    }
}

File: app/src/main/java/moe/tlaster/openween/core/model/favor/FavorTagModel.kt
package moe.tlaster.openween.core.model.favor

import com.fasterxml.jackson.annotation.JsonProperty


/**
 * Created by Tlaster on 2016/9/2.
 */
class FavorTagModel {
    @field:JsonProperty("id")
    var id: Long = 0
    @field:JsonProperty("tag")
    var tag: String? = null
}
